docs/Course Wiki/ECE Course Offerings/ECE417.md

```diff
@@ -0,0 +1,65 @@
+---
+tags:
+- Signal processing
+- Digital signal processing
+- Image processing
+---
+# ECE417
+
+ECE 417 (Multimedia Signal Processing) is a 4-credit-hour course that can be used to fulfill technical elective credit. It is only offered in fall semesters.
```

Comment:
Change to "is a 4-credit-hour course that satisfies the Technical Electives requirement for ECE majors."